demo/ContainerViewController.swift

```diff
@@ -187,6 +208,27 @@ extension ContainerViewController: ContainerPresenter {
         calibrationLabel.text = L10n.generalCalibration(Int(ceil(calibrationProgress.progress * 100)))
     }
     
+    func present(collisionObjectFrame: CGRect?, canvasSize: CGSize) {
+        guard let frame = collisionObjectFrame else {
+            collisitonObjectView.isHidden = true
+            collisionAlertView.isHidden = true
+            return
+        }
+        
+        let leftTop = frame.origin.convertForAspectRatioFill(from: canvasSize, to: view.bounds.size)
+        let rightBottom = CGPoint(x: frame.maxX, y: frame.maxY).convertForAspectRatioFill(from: canvasSize, to: view.bounds.size)
+        
+        let rect = CGRect(x: leftTop.x, y: leftTop.y, width: rightBottom.x - leftTop.x, height: rightBottom.y - leftTop.y)
+        let pct: CGFloat = 2.0
+        let newWidth: CGFloat = sqrt(rect.width * rect.width * pct)
```

Comment:
Is this is intended to be multiplying by some constant. If so should it involve `sqrt`?